librz/util/calc.c

```diff
@@ -133,6 +133,25 @@ static inline RzNumCalcValue Ndiv(RzNumCalcValue n, RzNumCalcValue v) {
 	return n;
 }
 
+static inline RzNumCalcValue Nexp(RzNumCalcValue n, RzNumCalcValue v) {
+	RzNumCalcValue exp_n = n;
+	if (v.d - (st64)v.n) {
+		RZ_LOG_WARN("floating point powers not yet supported\n");
+	}
+	if ((st64)v.n > 0) {
+		for (st64 i = 1; i < (st64)v.n; i++) {
+			n = Nmul(exp_n, n);
+		}
+	} else if ((st64)v.n < 0) {
+		for (st64 i = 1; i > (st64)v.n; i--) {
+			n = Ndiv(n, exp_n);
```

Comment:
I don't understand this... the first time it is executed it does `n / n`, which is ofc 1. I think it should start by doing `1 / n` and go on from there.